lib/modules/dosomething/dosomething_api/resources/member_resource.inc

```diff
@@ -90,6 +90,33 @@ function _member_resource_definition() {
       ),
     ),
     'targeted_actions' => array(
+      'update' => [
+        'file' => [
+          'type' => 'inc',
+          'module' => 'dosomething_api',
+          'name' => 'resources/member_resource',
+        ],
+        'help' => 'Update the users profile',
+        'args' => [
+          [
+            'name' => 'id',
+            'optional' => FALSE,
+            'source' => array('path' => 0),
+            'type' => 'string',
+            'description' => 'The Northstar ID of the user we\'re updating.',
```

Comment:
So the `/users` endpoint wasn't one I got a chance to update when I did the refactoring of the API using transformers, etc. I was only able to update what was immediately needed at the time for the mobile app. So the CRUD style operations might not be working because it's using the old approach and not fully set up to be RESTful.